lib/livebook_web/controllers/health_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule LivebookWeb.HealthController do
+  use LivebookWeb, :controller
+
+  def index(conn, _params) do
+    version = Application.spec(:livebook, :vsn) |> List.to_string()
+
+    json =
+      Jason.encode!(%{
+        "application" => "livebook",
+        "version" => version
+      })
+
+    send_resp(conn, 200, json)
```

Comment:
There is a json/2 helper in Phoenix that also sets the content type and encodes it for you!